copier/config/user_data.py

```diff
@@ -108,8 +112,38 @@ def cast_answer_type(answer: Any, type_fn: Callable) -> Any:
         return answer
 
 
+def render_value(value: Any, env: SandboxedEnvironment, context: AnyByStrDict) -> str:
+    """Render a single templated value using Jinja."""
+    try:
+        return env.from_string(value).render(**context)
+    except TypeError:
+        # value was not a string
+        return value
```

Comment:
Also maybe we should handle errors more explicitly:

```suggestion
    try:
        template = env.from_string(value)
    except TypeError:
        # value was not a string
        return value
    try:
        return template.render(**context)
```

I'm worried about the case that the template itself is badly programmed and raises a `TypeError` when rendered.